stubs/antlr4-python3-runtime/antlr4/CommonTokenFactory.pyi

```diff
@@ -0,0 +1,14 @@
+from _typeshed import Incomplete
+
+from antlr4.Token import CommonToken as CommonToken
+
+class TokenFactory: ...
+
+class CommonTokenFactory(TokenFactory):
+    DEFAULT: Incomplete
+    copyText: Incomplete
+    def __init__(self, copyText: bool = False) -> None: ...
+    def create(
+        self, source: tuple[Incomplete], type: int, text: str, channel: int, start: int, stop: int, line: int, column: int
```

Comment:
`tuple[Incomplete]` means "A tuple of length 1, where the sole element in the tuple is of type `Incomplete`". Is that what you meant, or did you want to say "A tuple of arbitrary length, in which all elements are of type `Incomplete`"? If the latter, you should do this instead:

```suggestion
        self, source: tuple[Incomplete, ...], type: int, text: str, channel: int, start: int, stop: int, line: int, column: int
```